anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc

```diff
@@ -520,7 +520,11 @@ For information on the redelivery policy, see xref:mule-runtime::redelivery-poli
 [%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Max Local Messages a| Number | Maximum number of messages to prefetch from the service and keep locally before dispatching to the flow | 30 |
+| Max Local Messages a| Number | Maximum number of messages to prefetch from the service and keep locally before dispatching to the flow.
+
+// MQ Prefetch maxLocalMessages
+include::connectors::partial$mq-disclaimer.adoc[tag=mqPrefetchMaxLocalMsgs]
+| 30 |
```

Comment:
how does moving |30|  below disclaimer look on the docs website?

Reply:
The |30| is the Default column in the table. Are you suggesting to move the note to the Required column?
![image](https://github.com/mulesoft/docs-connectors/assets/19573371/1a4871a0-8295-4931-a3bf-721e48d29476)

Reply:
No change necessary. I couldn't visualize the change earlier. This looks good. thanks!